src/components/Book.css

```diff
@@ -0,0 +1,78 @@
+.Book {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 700px;
+  background-color: #fff;
+  margin: 0 auto 15px auto;
+  padding: 15px 25px;
+}
+
```

Comment:
- [ ] Please add the bars that are in the book section. 

![tempsnip](https://user-images.githubusercontent.com/64173189/135726841-70a000ce-6726-4aee-8545-f70cdf5dbbb9.png)

😄